### PR TITLE
Added missing name property to BufferAttribute's typings.

### PR DESCRIPTION
**Pull request by mrschofield**
Hi,

The name property is missing from BufferAttribute.

Thanks!

### CODE REVIEW

**Comment by mrdoob** [comment]
Thanks!

File: src/core/BufferAttribute.d.ts [hunk 1]
```diff
@@ -6,6 +6,7 @@ export class BufferAttribute {
 	constructor( array: ArrayLike<number>, itemSize: number, normalized?: boolean ); // array parameter should be TypedArray.
 
 	uuid: string;
```
Comment (mrdoob) [inline src/core/BufferAttribute.d.ts]: I guess this line is wrong?

Comment (Mugen87) [reply src/core/BufferAttribute.d.ts]: Yes, `BufferAttribute` has no `uuid`.

Comment (Mugen87) [reply src/core/BufferAttribute.d.ts]: @mrschofield Would you like to correct this with another PR? 😇

Comment (mrschofield) [reply src/core/BufferAttribute.d.ts]: @Mugen87 done - https://github.com/mrdoob/three.js/pull/17019

**Pull request merged**